packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -186,6 +229,16 @@ const Updates = decorate([
           .sort()
           .map(name => `- ${name}: ${installedPackages[name]}`)
           .join('\n'),
+      xoaReleaseChannelsOptions: ({ xoaReleaseChannels }) => [
+        ...map(xoaReleaseChannels, elt => ({
+          label: elt.description,
+          value: elt.id,
+        })),
+        {
+          label: <span className='font-italic'>custom channel</span>,
```

Comment:
*custom* or *private*?
Be consistent.

Personnally I think *unlisted* is better. (/cc @pdonias)

Reply:
I think also that `unlisted` is better :+1: